constants/interface-routes/configs.json

```diff
@@ -1702,6 +1702,19 @@
 					"packageName": "elevate-project"
 				}
 			]
+		},
+		{
+			"sourceRoute": "/project/v1/project/templates/publishTemplateAndTasks",
```

Comment:
@priyanka-TL Let's create a new module SCP and then put all relevant APIs there.

Reply:
@aks30 done